src/components/guidance-content/help/HelpGeneral.html

```diff
@@ -0,0 +1,330 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
+<html>
```

Comment:
This page is huge. Can it be given a table of contents, ideally with links? I vaguely recall you saying something about anchor links not working, though, so maybe not.

Reply:
Love to, but time...

Reply:
https://github.com/pacificclimate/climate-explorer-frontend/issues/206